patches/tModLoader/Terraria/SceneMetrics.cs.patch

```diff
@@ -177,6 +177,18 @@
  		}
  
  		private void ExportTileCountsToMain() {
+@@ -355,6 +_,11 @@
+ 			PartyMonolithCount = _tileCounts[455];
+ 			GraveyardTileCount = _tileCounts[85];
+ 			GraveyardTileCount -= _tileCounts[27] / 2;
++
++			TileLoader.RecountTiles(this);
```

Comment:
Better to comment out the above code, if it's going to be ignored anyway

Reply:
This currently doesn't handle the `Main.remixWorld` condition changes which tiles count as jungle/evil/blood

Reply:
Oh, I was unaware of this condition. I'm not seeing it in the source code I have, is this in a newer version? (I forked from 1.4 default, if that helps anything)

Reply:
Ah, my apologies, I missed that this isn't targeting 1.4.4.

Development for 1.4 has stopped, so please rebase this onto the 1.4.4 branch.

Reply:
Understood, I'll do that. Should be a simple enough switch.